military-overlay/utils-source-and-test-data/schema-source-files/app6b/military_feature_schemas/Fields_Activities.csv

```diff
@@ -0,0 +1,30 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_setsubtypes, field_subtype, field_notes
+affiliation,String,25,Affiliation,NULLABLE,Affiliation,,FALSE,,
+extendedfunctioncode,String,150,Extended Function Code,NULLABLE,Activities,,FALSE,,
+echelonmobility,String,30,Echelon/Mobility,NULLABLE,EchelonMobility,,FALSE,,
+status,String,20,Status,NULLABLE,Statuses,,FALSE,,
+hqtffd,String,30,HQ/TF/FD,NULLABLE,HQ_TF_FD,,FALSE,,
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,FALSE,,
+additionalinformation,String,20,Additional Information,NULLABLE,,,FALSE,,
+staffcomment,String,20,Staff Comments,NULLABLE,,,FALSE,,
+higherformation,String,21,Higher Formation,NULLABLE,,,FALSE,,
+reinforced,SmallInteger,2,Reinforced or Reduced,NULLABLE,Reinforced,,FALSE,,
+combateffectiveness,String,5,Combat Effectiveness,NULLABLE,CombatEffectiveness,,FALSE,,
+credibility,SmallInteger,2,Credibility,NULLABLE,Credibility,,FALSE,,
+reliability,String,2,Reliability,NULLABLE,Reliability,,FALSE,,
+iff_sif,String,15,IFF/SIF/AIS,NULLABLE,,,FALSE,,
```

Comment:
@jeconley - I remember you IM'd me about this attribute "idmode" (2525) vs. "iff_sif" (app6b) for app6b. It is **iff_sif** in this schema - please let us know if it (or anything else you notice here) should be something different - thanks